src/components/calcite-time-picker/calcite-time-picker.scss

```diff
@@ -0,0 +1,154 @@
+:host {
+  @apply text-color-1
+    inline-block
+    font-medium
+    select-none;
+}
+
+.time-picker {
+  @apply shadow-2
+    items-center
+    flex;
+  border-radius: var(--calcite-border-radius);
+}
+
+span {
```

Comment:
I think because there are so many spans, both for the number values and the delimiter spans, that I think this selector is still the most appropriate way to select things.  Adding a class would just add unnecessary complexity.